controllers/article-controller.js

```diff
@@ -0,0 +1,29 @@
+const { fetchArticleById } = require("../models/article-model");
+
+const getArticleById = (request, response, next) => {
+  const { article_id } = request.params;
+
+  if (/[A-Za-z]/g.test(article_id)) {
+    return response
```

Comment:
We can handle this error much more simply through PSQL. Inputting a string instead of a number would violate the table's rules which would throw a PSQL error. We could catch that and check the code. This helps us keep all our error handling in error handlers.